Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
 
 
 Compiler Features:
+ * EVM: Introduce ``prague`` as an evm version option.
```

Comment:
```suggestion
 * EVM: Support for the EVM version "Prague".
```

Reply:
But wouldn't the previous change be more accurate? I mean, we don't really support `Prague` yet, just introduced the evm version option in the compiler.